app/scripts/modules/appengine/serverGroup/configure/wizard/basicSettings.controller.ts

```diff
@@ -32,6 +35,26 @@ class AppengineServerGroupBasicSettingsCtrl implements IController {
     }
   }
 
+  public isGitSource(): boolean {
+    return this.$scope.command.sourceType === AppengineSourceType.GIT as string;
```

Comment:
I am very surprised you have to cast this but I believe you.

Reply:
Removed.